tests/functional/provider_client/test_onboarding_token_generation.py

```diff
@@ -0,0 +1,103 @@
+import logging
+import pytest
+import time
+
+from ocs_ci.ocs.resources import pod
+from ocs_ci.ocs import constants, ocp
+from ocs_ci.framework import config
+from ocs_ci.framework.testlib import (
+    skipif_ocs_version,
+    ManageTest,
+    tier1,
+    skipif_ocp_version,
+    skipif_managed_service,
+    runs_on_provider,
+    skipif_external_mode,
+    green_squad,
+)
+from ocs_ci.ocs.ui.validation_ui import ValidationUI
+
+log = logging.getLogger(__name__)
+
+
+@tier1
+@green_squad
+@skipif_ocs_version("<4.15")
+@skipif_ocp_version("<4.15")
+@skipif_external_mode
+@runs_on_provider
+@skipif_managed_service
+class TestOnboardingTokenGeneration(ManageTest):
+    @pytest.fixture(autouse=True)
+    def setup(self, request):
+        """
+        Resetting the default value of KeyRotation
```

Comment:
Any code missing in this method for resetting?

Reply:
Addressed in latest commit.